### PR TITLE
install tools

### PR DESCRIPTION
**Pull request by achmelo**
Build and publish attls library.

### CODE REVIEW

File: attls/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,88 @@
+def serverCredentials = []
+def ARTIFACTORY_CREDENTIALS_ID = 'zowe.jfrog.io'
+Map TEST_SERVERS = [
+        'marist'  : [
+                ansible_host     : 'marist-1',
+                ssh_hostport     : 'ssh-marist-server-zzow01-hostport',
+                ssh_userpass     : 'ssh-marist-server-zzow01',
+                node_home_pattern: '/ZOWE/node/node-{NODE_VERSION}-os390-s390x'
```
Comment (balhar-jakub) [inline attls/Jenkinsfile:8]: Node is ignored on the other two servers or use the same?

File: attls/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    ))
+}
+pipeline {
+
+    agent {
+        label 'zowe-jenkins-agent'
+    }
+
+    parameters {
+        booleanParam(name: 'RELEASE', defaultValue: false, description: 'Toggle this to publish release')
```
Comment (balhar-jakub) [inline attls/Jenkinsfile:41]: Is there a difference between Patch, Minor and Major release?

**Review by balhar-jakub — APPROVED** [review]

File: attls/Jenkinsfile (hunk 1 above)
Comment (achmelo) [reply attls/Jenkinsfile:8]: I took the map from zowe install packaging so I am not sure what is the actual situation

File: attls/Jenkinsfile (hunk 2 above)
Comment (achmelo) [reply attls/Jenkinsfile:41]: my intention was to increase patch only for now as I dont expect any serious development for this library other than occasional patches

Comment (balhar-jakub) [reply attls/Jenkinsfile:41]: Ok, we can for now, but I expect that we will pull in other zOS related code.

**Pull request merged**